rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -92,56 +101,65 @@ Time zone for defining downtimes.
 Click *Create* to add a downtime to the application.
 . Click *Save*.
 
-Learn how to xref:myrpa-start.adoc#required-process-applications[assign required applications to a process].
-
-Learn how to xref:botmanagement-manage.adoc#bot-assign-servicetimes-applications[assign required applications to a bot].
-
 ==== Edit an Application
 
 Edit an application to change its data or down-times:
 
-. In the *Process Applications* view of the *Process Automation* module, click *Create*.
+. Open the *Process Applications* view of the *Process Automation* module.
+. Click *Create*.
 . Edit the form *Edit the application*.
++
+The properties are explained in the form <<form-create-process-applictions, *Create a new application*>>.
```

Comment:
What does it mean that the properties are explained in a form? Do you mean a topic?

Reply:
paragraph in topic explaining the form -> replaced with 'The properties are explained in the topic <<form-create-process-applictions, *Create an Application*>>.'